code/InterestAR/Assets/Scripts/Services/GoogleMapsService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class GoogleMapsService
+{
+
+    public List<Place> GetCoördinates(string latitude, string longitude, int radius)
```

Comment:
Ik zou proberen of je deze methode nog zo async mogelijk kan maken. Dit lost misschien het vertraagd opstarten probleem op.